### PR TITLE
[INTEGRATION][dbt] add progressbar for sending OL events

### PR DESCRIPTION
**Pull request by mobuchowski**
I've used [tqdm library](https://github.com/tqdm/tqdm) to add progress bar.

Result: https://asciinema.org/a/GSTVkQA1ZEJV1AFPiCTOkIjiJ

Signed-off-by: Maciej Obuchowski <maciej.obuchowski@getindata.com>

### CODE REVIEW

File: integration/dbt/scripts/dbt-ol [hunk 1]
```diff
@@ -53,9 +54,10 @@ def main():
 
     events = processor.parse().events()
 
-    for event in events:
+    for event in tqdm(events, desc="Emitting OpenLineage events:"):
+        time.sleep(0.25)
```
Comment (julienledem) [inline integration/dbt/scripts/dbt-ol]: why do we need to sleep here?

File: integration/dbt/scripts/dbt-ol [hunk 2]
```diff
@@ -53,9 +54,10 @@ def main():
 
     events = processor.parse().events()
 
-    for event in events:
+    for event in tqdm(events, desc="Emitting OpenLineage events:"):
```
Comment (julienledem) [inline integration/dbt/scripts/dbt-ol:57]: it looks like ":" is not needed here. Looking at your recording there are two in the display

File: integration/dbt/scripts/dbt-ol (hunk 1 above)
Comment (julienledem) [reply integration/dbt/scripts/dbt-ol]: was this so that the recording looks nicer? :)

File: integration/dbt/scripts/dbt-ol [hunk 3]
```diff
@@ -53,9 +54,10 @@ def main():
...
+        time.sleep(0.25)
         client.emit(event)
-    print(f"Emitted {len(events)} openlineage events")
+    print(f"Emitted {len(events)} OpenLineage events")
```
Comment (julienledem) [inline integration/dbt/scripts/dbt-ol:59]: can you print a message at the very beginning that mentions OpenLineage?
- the version (just like dbt does)
- that we're going to send events at the end.

**Review by julienledem — CHANGES_REQUESTED** [review]
this progress bar looks great!

File: integration/dbt/scripts/dbt-ol (hunk 1 above)
Comment (mobuchowski) [reply integration/dbt/scripts/dbt-ol]: Yes :wink: 
I've removed it minute before you looked.

File: integration/dbt/scripts/dbt-ol (hunk 3 above)
Comment (mobuchowski) [reply integration/dbt/scripts/dbt-ol:62]: Done

**Review by julienledem — APPROVED** [review]

**Pull request merged**